README.md

```diff
@@ -1,2 +1,5 @@
+[![Build status](https://ci.appveyor.com/api/projects/status/apy6236i4ytwcldh?svg=true)](https://ci.appveyor.com/project/kubmir/kentico-onboarding-cs)
+[![Build status](https://ci.appveyor.com/api/projects/status/apy6236i4ytwcldh/branch/develop?svg=true&passingText=develop%20-%20OK)](https://ci.appveyor.com/project/kubmir/kentico-onboarding-cs/branch/develop)

```

Comment:
These badges are really pretty, but you have 2 of them next to each other without any distinction what seems a bit confusing to me. On the other hand, never mind me: https://github.com/Microsoft/vscode

Reply:
If build of develop branch will pass there will be a message in the green part of the second badge - develop - OK. The left one will be same as now.